dotCMS/src/main/java/com/dotmarketing/osgi/OSGIProxyServlet.java

```diff
@@ -89,4 +96,20 @@ private BundleContext getBundleContext () throws ServletException {
         throw new ServletException( "Bundle context attribute [" + BundleContext.class.getName() + "] not set in servlet context" );
     }
 
+    private static void setIsInitialized(Boolean initialized) {
+        isInitialized = initialized;
+    }
+
+    private static void setServletConfig(ServletConfig config) {
+        servletConfig = config;
+    }
+
+    private static void setBundleContext(BundleContext context) {
+        bundleContext = context;
+    }
+
+    private static void setTracker(DispatcherTracker dispatcherTracker) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "DispatcherTracker"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)